config/environments/sandbox.rb

```diff
@@ -1 +1,5 @@
 require Rails.root.join("config/environments/production")
+
+Rails.application.configure do
+  config.trs_write_only = true
+end
```

Comment:
Can we control this via an environment variable? We have various other settings like that too and they're easily findable/configurable in [sandbox.yml](https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/config/terraform/application/config/sandbox.yml)

Reply:
Yeah sure 👍